README.rst

```diff
@@ -1,59 +1,10 @@
-<img src="docs/source/logo/BluePyOptBanner.png"/>
-
------------------
-
-# BluePyOpt: Blue Brain Python Optimisation Library
-
-<table>
-<tr>
-  <td>Latest Release</td>
-  <td>
-    <a href="https://pypi.org/project/bluepyopt/">
-    <img src="https://img.shields.io/pypi/v/bluepyopt.svg" alt="latest release" />
-    </a>
-  </td>
-</tr>
-<tr>
-  <td>Documentation</td>
-  <td>
-    <a href="https://bluepyopt.readthedocs.io/">
-    <img src="https://readthedocs.org/projects/bluepyopt/badge/?version=latest" alt="latest documentation" />
-    </a>
-  </td>
-</tr>
-<tr>
-  <td>License</td>
-  <td>
-    <a href="https://github.com/BlueBrain/bluepyopt/blob/master/LICENSE.txt">
-    <img src="https://img.shields.io/pypi/l/bluepyopt.svg" alt="license" />
-    </a>
-</td>
-</tr>
-<tr>
-  <td>Build Status</td>
-  <td>
-    <a href="https://github.com/BlueBrain/BluePyOpt/actions">
-    <img src="https://github.com/BlueBrain/BluePyOpt/workflows/Build/badge.svg?branch=master" alt="Actions build status" />
-    </a>
-  </td>
-</tr>
-<tr>
-  <td>Coverage</td>
-  <td>
-    <a href="https://codecov.io/gh/BlueBrain/bluepyopt">
-    <img src="https://codecov.io/github/BlueBrain/BluePyOpt/coverage.svg?branch=master" alt="coverage" />
-    </a>
-  </td>
-</tr>
-<tr>
-	<td>Gitter</td>
-	<td>
-		<a href="https://gitter.im/bluebrain/bluepyopt">
-		<img src="https://badges.gitter.im/Join%20Chat.svg"
-	</a>
-	</td>
-</tr>
-</table>
+|banner|
+
+|release| |docs| |license| |build| |coverage| |gitter|
```

Comment:
Please leave the original table here. This was based on feedback from a user, who didn't notice the banners.